cloup/_sections.py

```diff
@@ -122,13 +122,13 @@ def __init__(
         :param kwargs:
             keyword arguments forwarded to the next class in the MRO
         """
+        super().__init__(*args, commands=commands, **kwargs)  # type: ignore
```

Comment:
This change probably makes sense but as innocent as it may look, it's a breaking change lol. I'm not sure why I decided to call super() at the end but this change should be done on a separate PR.

Reply:
@janluke Yeah I worried about this... However, this change doesn't cause any tests to fail. The reason I moved it is because it's needed to set `self.chain` before the rest of the code runs — which expects the `chain` attribute to exist.

Reply:
Can you open a separate PR with this explanation, please?

Reply:
Opened an issue about it: 
- #118